script/validate-data/index.ts

```diff
@@ -77,17 +87,28 @@ async function checkWorkflow(workflowPath: string, propertiesPath: string): Prom
 
     const propertiesFileContent = await fs.readFile(propertiesPath, "utf8")
     const properties: WorkflowProperties = JSON.parse(propertiesFileContent)
-
+    if(properties.name && properties.name.trim.length > 0) {
+      workflowErrors.name = properties.name
+    }
     let v = new validator();
     const res = v.validate(properties, propertiesSchema)
     workflowErrors.errors = res.errors.map(e => e.toString())
-
-    if (properties.iconName && !properties.iconName.startsWith("octicon")) {
-      try {
-        await fs.access(`../../icons/${properties.iconName}.svg`)
-      } catch (e) {
-        workflowErrors.errors.push(`No icon named ${properties.iconName} found`)
+    
+    if (properties.iconName) {
+      if(!properties.iconName.startsWith("octicon ")) {
```

Comment:
Should we use regex to handle one or more spaces after `octicon`?
and should we handle it with lower casing, like `properties.iconName.tolower.startsWith`?

Reply:
- Used regex for checking the string.
- The value is case sensitive as seen [here](https://github.com/github/github/blob/fb08a1cd44254fa796fac532afb1f9d55c9e82f1/app/views/repository_actions/_workflow_template.html.erb#L44)